scripts/netContain/release_image.sh

```diff
@@ -1,25 +1,26 @@
 #!/bin/bash
 
+set -euo pipefail
+
 contiv_version=""
 docker_user="contiv"
```

Comment:
Done, also removed the "password" option and removed username/password from getopt